src/gui/editorwidgets/qgsrelationreferenceconfigdlg.h

```diff
@@ -39,6 +39,13 @@ class GUI_EXPORT QgsRelationReferenceConfigDlg : public QgsEditorConfigWidget, p
     QVariantMap config() override;
     void setConfig( const QVariantMap &config ) override;
 
+  public slots:
+
+    /**
+     * Opens an expression dialog and sets its value as filter expression for the relation reference.
+     */
+    void editExpression();
```

Comment:
Can this be a private slot?

Reply:
Got inspired by `QgsValueRelationConfigDlg::editExpression`, which is a public slot:
https://github.com/qgis/QGIS/blob/65eed59ed0a0ff4fc27d332bd6124c0d2ad69a4f/src/gui/editorwidgets/qgsvaluerelationconfigdlg.h#L39-L40

Reply:
Looks like a leftover, only `public` if required.

Reply:
I will make it `private` then, no requirements involved.